pom.xml

```diff
@@ -175,7 +175,7 @@
     <!-- Leaving libfb303 at 0.9.3 regardless of libthrift: As per THRIFT-4613 The Apache Thrift project does not publish items related to fb303 at this point -->
     <libfb303.version>0.9.3</libfb303.version>
     <libthrift.version>0.16.0</libthrift.version>
-    <log4j2.version>2.18.0</log4j2.version>
+    <log4j2.version>2.23.1</log4j2.version>
```

Comment:
Can we use the lateset log4j2 version? 2.24.1
https://github.com/apache/logging-log4j2/releases/tag/rel%2F2.24.1

Reply:
- I have submitted new commits, just waiting for CI to complete first. 
- Of course, I will say that hive's PR template requires that PRs that update dependencies need to submit a separate `dependency-tree.txt` file, which is a pain. Because my IDE and shell are installed under WSLg, and I need to use Windows to search for files in the remote file system to upload `dependency-tree.txt`. If hive can integrate with ASF's https://ge.apache.org/, I think uploading a suspicious `dependency-tree.txt` will no longer be necessary. Because there are more straightforward interfaces like https://ge.apache.org/s/sztfzljasgv52/dependencies that output dependency trees.